src/features/events/entities/virtualEvent.entity.ts

```diff
@@ -0,0 +1,24 @@
+import { Column, Entity, OneToOne, PrimaryGeneratedColumn } from "typeorm";
+import { EventEntity } from "./event.entity";
+
+export enum EventType {
+    zoom
+}
+
+@Entity()
+export class VirtualEvent {
+    @PrimaryGeneratedColumn()
+    virtual_id: number;
```

Comment:
please use the CamelCase naming convention,

Instead of virtual_id use `virtualId`. But here as primary key, just use `id` as name